bot/cogs/tags.py

```diff
@@ -1,14 +1,16 @@
 import logging
 import re
 import time
+from asyncio import TimeoutError
```

Comment:
I suggest changing this to
```py
import asyncio
```
then you can do
```py
except asyncio.TimeoutError
```
To avoid shadowing the base `TimeoutError`.